plugins/filetools/plugin.js

```diff
@@ -885,3 +911,16 @@
  * @cfg {String} [fileTools_defaultFileName='']
  * @member CKEDITOR.config
  */
+
+/**
+ * Additional headers of XMLHttpRequest used during file upload with plugins: {@link CKEDITOR.fileTools} and filebrowser.
+ *
+ * 	config.xmlHttpRequestHeaders = {
+ * 		'Cache-Control': 'no-cache',
+ * 		'X-CUSTOM': 'HEADER'
+ * 	};
+ *
+ * @since 4.8.1
+ * @cfg {Object} [xmlHttpRequestHeaders=null]
```

Comment:
This config property should be prefixed with `fileTools_` just like the other filetools config property.